styles.css

```diff
@@ -0,0 +1,95 @@
+*{
+    box-sizing: border-box;
+    margin:0;
+}
+body{
+    background-color: hsl(30, 38%, 92%);
+    font-size: 14px;
+    font-family: 'Montserrat', sans-serif;
+}
+.cardContainer{
+    background-color: white;
+    width:700px;
+    height:500px;;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 1rem;
+    display:flex;
+    flex-direction: row;
+    overflow: hidden;
+}
+.imageContainer{
+    background: url(./images/image-product-desktop.jpg);
+    background-size: cover;
+    width:50%;
+    height:100%;
+}
+
+.infoContainer{
+    display:flex;
+    flex-direction:column;
+    justify-content: space-evenly;
+    width:50%;
+    height:100%;
+    padding:2rem;
+    color:hsl(228, 12%, 48%);
+}
+.infoContainer h1{
+    font-family: 'Fraunces', serif;
+    color:black;
+}
+.productType{
+    font-weight: 700;
+}
+.pricesContainer{
+    display:flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 30px;
+}
+.presentPrice{
+    color:hsl(158, 36%, 37%);
+    font-size: 1.8rem;
+    font-family: 'Fraunces', serif;
+    font-weight: 700;
+}
+.previousPrice{
+    color:hsl(228, 12%, 48%);
+    text-decoration: line-through;
+    align-self: center;
+}
+.addToCartButton{
+    background-color:hsl(158, 36%, 37%);
+    color:white;
+    font-weight: 700;
+    height:2.5rem;
+    border:none;
+    border-radius:0.3rem;
+}
+.addToCartButton img{
+    vertical-align: middle;
+}
+.addToCartButton:hover{
+    cursor: pointer;
+    background-color:darkgreen;
```

Comment:
dobrą praktyką (chociaż nie tak często i chętnie stosowaną) jest używanie jednego stylu zapisu koloru w całej aplikacji - nie jest to must have, ale dobry zwyczaj